README.md

```diff
@@ -0,0 +1,36 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse!
+
+> Description the project.
+
+
+## Built With 
+
+- HTML
+- CSS
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```

Comment:
- [ ] [REQUIRED]  Please add descriptive informations to `line 5` and `Author` section. And please implement social networks links.